File: cardView/app/src/main/java/com/example/cardview/MainActivity.java
package com.example.cardview;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.Button;
import android.widget.EditText;

public class MainActivity extends AppCompatActivity {
EditText e1,e2;
Button b1;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
       e1=findViewById(R.id.e1);
        e2=findViewById(R.id.e2);
        b1=findViewById(R.id.b1);
    }
}

File: connectfirebase/app/src/main/java/com/example/connectfirebase/MainActivity.kt
package com.example.connectfirebase

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.EditText

   public class MainActivity extends AppCompatActivity {
EditText txtname,txtage,txtphone,txtheight;
    Button btnsave;
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main)
         txtname=(EditText)findViewId(R.id.txtname)


    }
}
